app/views/groups/index.html.erb

```diff
@@ -1,14 +1,22 @@
-<p style="color: green"><%= notice %></p>
-
-<h1>Groups</h1>
-
-<div id="groups">
+<% if user_signed_in? %>
   <% @groups.each do |group| %>
-    <%= render group %>
-    <p>
-      <%= link_to "Show this group", group %>
-    </p>
+    <div class="card d-flex m-5 p-3">
+      <%= link_to group_path(group.id), class: "font-light" do %>
+        <div class="d-flex align-items-center justify-content-between">
+          <div class="d-flex align-items-center ms-4 me-5">
+            <%= image_tag group.icon, class: "image-size m-3" %>
```

Comment:
- [ ] Good job so far 👏 Great improvements **based on the previous reviewer. But as he suggested to match the design given,** the images on your design are way too small compared to the ones in the design given. Please look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/221211781-69e7ce4e-b183-43a7-ae3b-a0af40d38c77.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/221211882-870ac9b8-e8d6-4dae-a707-b51542f70abc.png)

So please increase their size so that at least they can be visible 😃 👍